html/js/graph.js

```diff
@@ -0,0 +1,1044 @@
+// Declaration of d3 variable (cf. d3js)
+var d3;
+
+// **************
+// constants_endpoints
+// **************
+// Endpoints (cf. REST endpoints in App.py)
+var URL_ROOT_API = "https://tuttifrutti.alwaysdata.net/";
```

Comment:
@stephanegigandet @teolemon Is it ok to query an external API from openfoodfacts.org and it's sister domains? I'm not sure about privacy/data security policies etc.

Reply:
What's behind the API? Is it something that would be easy to host ourselves?

Reply:
@stephanegigandet @teolemon @hangy 
This URL_ROOT_API is used in 4 parts:
![image](https://user-images.githubusercontent.com/15727915/92410896-fe514d80-f145-11ea-845d-4357f2ee5a43.png)
* to access countries.json file (not sure anymore how I got it, but probably on OFF somehow) : can be hosted (or is already hosted?) in OFF-domain -> https://github.com/oricdev/off_product_comparator/blob/master/static/data/countries.json
* to access stores based on the selected country: stores are retrieved from OFF site and then stored locally (see Python-code in screenshot) -> can be computed in JS in order to fetch correct stores-file
![image](https://user-images.githubusercontent.com/15727915/92411000-6f910080-f146-11ea-9006-b0cadd048a7d.png)
* to access the score details json file: it could be added to OFF and also simplified by removing all non-relevant items .. please check https://github.com/oricdev/off_product_comparator/blob/master/static/data/db_stats.json
* fetchPGraph is compulsory as long as the db holding matching computations is externalized in tuttifrutti website.. code of fetchPGraph is here https://github.com/oricdev/off_product_comparator/blob/master/app.py starting on line 141

Among these 4 items, fetchPGraph may be the only one which is compulsory so far. Please advise...